bool ready
---
geometry_msgs/Pose current_target_position